system/utils.py

```diff
@@ -806,6 +808,82 @@ def get_node_did(node_alias, pool_info=None):
     return is_exception_raised
 
 
+# TODO remove this eventually and all custom eventuallies above later
```

Comment:
I don't see any usage of that helper? Do you think it is necessary in scope of the current PR?

Reply:
Removed.